winmake.ps1

```diff
@@ -28,6 +28,37 @@ function Local-Machine {
     Run-Command "./test/tools/build/ginkgo.exe -vv  --tags `"$remotetags`" -timeout=90m --trace --no-color $files pkg/machine/e2e/. "
 }
 
+function Win-SSHProxy {
+    # git is not installed by default on windows
+    # fail if git doesn't exist
+    Get-Command git  -ErrorAction SilentlyContinue  | out-null
+    if(!$?){
+        Write-Host "Git not installed, cannot build Win-SSHProxy"
+        Exit 1
+    }
+    # Update the SHA in Makefile as well
+    $GV_GITURL = "https://github.com/containers/gvisor-tap-vsock.git"
+    $GV_SHA = "db608827124caa71ba411cec8ea959bb942984fe"
+
+    if (Test-Path ./tmp-gv) {
+        Remove-Item ./tmp-gv -Recurse -Force -Confirm:$false
+    }
+    New-Item  ./tmp-gv -ItemType Directory -ea 0
+    Push-Location ./tmp-gv
+    Run-Command "git init"
+    Run-Command "git remote add origin $GV_GITURL"
+    Run-Command "git fetch --depth 1 origin $GV_SHA"
+    Run-Command "git checkout FETCH_HEAD"
+    Run-Command "go build -ldflags -H=windowsgui -o bin/win-sshproxy.exe ./cmd/win-sshproxy"
+    Run-Command "go build -ldflags -H=windowsgui -o bin/gvproxy.exe ./cmd/gvproxy"
+    Pop-Location
+
+    New-Item -ItemType Directory -Force -Path "./bin/windows"
+    Copy-Item -Path "tmp-gv/bin/win-sshproxy.exe" -Destination "bin/windows/"
+    Copy-Item -Path "tmp-gv/bin/gvproxy.exe" -Destination "bin/windows/"
```

Comment:
Consider adding a comment: Is this destination "magic" or significant somehow - like on the default `$PATH` or expected by some packaging tool?